presign/base/models.py

```diff
@@ -407,11 +408,29 @@ def change_state(self, action):
         self.state = next_state
         self.save(update_fields=["state"])
 
-    def send_change_state_email(self, request, action):
+        # if the action is not a answer save, we set all current messages to inactive
+        if action != ParticipantStateActions.ANSWERS_SAVED:
```

Comment:
I would prefer to pass a flag to the function to trigger this (or something like `state_caused_by = 'PARTICIPANT' | 'ORGANIZER'`)

Reply:
Do we really need this? As of our discussion yesterday, this is relative straightforward (we don't wanna expire messages on saved answers, all other cases are okay)